Resources/Prototypes/Guidebook/rules.yml

```diff
@@ -1,15 +1,20 @@
-﻿- type: guideEntry # Default for forks and stuff. Should not be listed anywhere if the server is using a custom ruleset.
-  id: DefaultRuleset
-  name: guide-entry-rules
-  ruleEntry: true
-  text: "/ServerInfo/Guidebook/ServerRules/DefaultRules.xml"
+# CD: This entire file has been changed
+
+# CD: Disable wizden rules
+#- type: guideEntry # Default for forks and stuff. Should not be listed anywhere if the server is using a custom ruleset.
+#  id: DefaultRuleset
+#  name: guide-entry-rules
+#  ruleEntry: true
+#  text: "/ServerInfo/Guidebook/ServerRules/DefaultRules.xml"
 
 - type: guideEntry
   id: CosmaticDriftRules
   name: guide-entry-rules-cd
   ruleEntry: true
   priority: 0
   text: "/ServerInfo/Guidebook/ServerRules/CosmaticDriftRules.xml"
+  children:
+  - CDSop
```

Comment:
Right now I have it under Rules. Do you think that is a good place to put it?

Reply:
Yes and no. It's rules to security and command, but it could cause confusion to anyone else it isn't part of the rules to (the majority), it is also an in-character document, which rules imply it is not. So perhaps we need a section for "In-character documentation" and place it under that.

Reply:
Good point, I will move it tomorrow